pykwalify/rule.py

```diff
@@ -380,9 +380,9 @@ def init(self, schema, path):
                     t = DEFAULT_TYPE
                     self.type = t
             else:
-                if not isinstance(schema["type"], str):
+                if not (isinstance(schema["type"], str) or isinstance(schema["type"], unicode)):
```

Comment:
You glob them together into into something like
```
if not isinstance(schema["type"], (str, int))
```
and get the same logic and much cleaner code.